### PR TITLE
Feature/appmodule

### PR DESCRIPTION
**Pull request by chr1s4ng3l**
App Module Implementation

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -34,11 +36,21 @@ android {
...
     implementation 'androidx.appcompat:appcompat:1.6.0'
     implementation 'com.google.android.material:material:1.8.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
+    implementation 'com.squareup.moshi:moshi-kotlin:1.14.0'
+
+    implementation 'com.google.code.gson:gson:2.10.1'
+    implementation 'com.squareup.retrofit2:converter-gson:2.9.0'
+    implementation 'com.squareup.retrofit2:retrofit:2.9.0'
+
+    implementation 'com.google.dagger:dagger-android:2.44.2'
```
Comment (sarisiza) [inline app/build.gradle]: Aren't we using Hilt instead of Dagger?

File: app/build.gradle [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
     id 'com.android.application'
     id 'org.jetbrains.kotlin.android'
+
+
 }
 
 android {
     namespace 'com.example.thecocktailsapp'
-    compileSdk 32
+    compileSdk 33
```
Comment (sarisiza) [inline app/build.gradle]: I believed this was already merged on PR#1

File: app/src/main/java/com/example/thecocktailsapp/CocktailApp.kt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package com.example.thecocktailsapp
```
Comment (sarisiza) [inline app/src/main/java/com/example/thecocktailsapp/CocktailApp.kt:1]: This was already merged in PR#1

File: app/src/main/java/com/example/thecocktailsapp/model/CocktailIInfo/CocktailInfo.kt [hunk 4]
```diff
@@ -0,0 +1,11 @@
+package com.example.thecocktailsapp.model.CocktailIInfo
```
Comment (sarisiza) [inline app/src/main/java/com/example/thecocktailsapp/model/CocktailIInfo/CocktailInfo.kt]: This was already merged in PR#1

File: app/src/main/java/com/example/thecocktailsapp/model/CocktailIInfo/DrinkXX.kt [hunk 5]
```diff
@@ -0,0 +1,111 @@
+package com.example.thecocktailsapp.model.CocktailIInfo
```
Comment (sarisiza) [inline app/src/main/java/com/example/thecocktailsapp/model/CocktailIInfo/DrinkXX.kt]: This was already merged in PR#1

File: app/src/main/java/com/example/thecocktailsapp/model/CocktailsByName/CocktailsByName.kt [hunk 6]
```diff
@@ -0,0 +1,11 @@
+package com.example.thecocktailsapp.model.CocktailsByName
```
Comment (sarisiza) [inline app/src/main/java/com/example/thecocktailsapp/model/CocktailsByName/CocktailsByName.kt]: This was already merged in PR#1

File: app/src/main/java/com/example/thecocktailsapp/model/CocktailsByName/Drink.kt [hunk 7]
```diff
@@ -0,0 +1,111 @@
+package com.example.thecocktailsapp.model.CocktailsByName
```
Comment (sarisiza) [inline app/src/main/java/com/example/thecocktailsapp/model/CocktailsByName/Drink.kt]: This was already merged in PR#1

File: app/src/main/java/com/example/thecocktailsapp/model/RandomCocktail/DrinkX.kt [hunk 8]
```diff
@@ -0,0 +1,15 @@
+package com.example.thecocktailsapp.model.RandomCocktail
```
Comment (sarisiza) [inline app/src/main/java/com/example/thecocktailsapp/model/RandomCocktail/DrinkX.kt]: This was already merged in PR#1

File: app/src/main/java/com/example/thecocktailsapp/model/RandomCocktail/RandomCocktails.kt [hunk 9]
```diff
@@ -0,0 +1,11 @@
+package com.example.thecocktailsapp.model.RandomCocktail
```
Comment (sarisiza) [inline app/src/main/java/com/example/thecocktailsapp/model/RandomCocktail/RandomCocktails.kt]: This was already merged in PR#1

**Review by sarisiza — CHANGES_REQUESTED** [review]
Most of these files were already merged from PR#1 
Please merge and update

**Review by sarisiza — APPROVED** [review]
LGTM!
Just update the Trello to add a card for this

**Review by luizeth0 — APPROVED** [review]

**Review by inaki92 — APPROVED** [review]
lgtm!

**Pull request merged**